doc/user_manual/optimizer.tex

```diff
@@ -166,7 +186,10 @@ \subsubsection{Simultaneous Perturbation Stochastic Approximation (SPSA)}
     or minimization. Available options are \xmlString{max} and \xmlString{min}.
     \default{Min};
     \item \xmlNode{thresholdTrajRemoval}, \xmlDesc{float, optional field}, this will be used to determine the convergence of different
-    optimization trajectories when multiple trajectories is handled by \xmlNode{SPSA}.
+    optimization trajectories on each other when multiple trajectories is handled by \xmlNode{SPSA}.  When one
+    trajectory comes within tolerance of a point on another trajectory, the first will be removed in interest
+    of the second.  Note that this value is
+    calculated as Euclidean distance in a normalized 0 to 1 cubic domain, not the original input space domain.
```

Comment:
That's a decent idea.  I would like to see some user feedback on this functionality to gain an idea of what they intuit the meaning of "remove a redundant trajectory" to be.